### PR TITLE
Chore/release candidate

### PR DESCRIPTION
**Pull request by SethDavenport**
Continuing from #72 - updated the example app to use Angular 2 release candidate 0.

### CODE REVIEW

**Comment by TobiasKrogh** [comment]
CI seems failing again due to a stale npm cache

@SethDavenport for what is the `@angular/compiler` needed? it is not used in the example app as far as I can see

File: package.json [hunk 1]
```diff
@@ -50,10 +50,10 @@
     "ts-loader": "^0.8.1",
     "ts-node": "^0.5.5",
     "typescript": "^1.8.10",
-    "angular2": "2.0.0-beta.17",
+    "@angular/core": "2.0.0-rc.0",
```
Comment (nexus-uw) [inline package.json]: rc1 was released late yesterday (https://github.com/angular/angular/blob/master/CHANGELOG.md#200-rc1-2016-05-03)

**Comment by SethDavenport** [comment]
Hi Tobias, 

@angular/compiler is a peer dependency for @angular/platform-browser-dynamic, which the example is using.

As for CI, yeah - circle doesn't handle peer dependency updates very well due to the way it caches `node_modules`.  I did a 'rebuild without cache' and it's now fine.

**Comment by SethDavenport** [comment]
Updated to RC1.

**Pull request merged**